lib/identity_config.rb

```diff
@@ -270,7 +270,9 @@ def self.build_store(config_map)
     config.add(:remember_device_expiration_hours_aal_1, type: :integer)
     config.add(:remember_device_expiration_hours_aal_2, type: :integer)
     config.add(:report_timeout, type: :integer)
+    config.add(:requests_per_ip_cidr_allowlist, type: :comma_separated_string_list)
```

Comment:
is the CIDR part of this same PR or a different one?

Reply:
Planning to add it here, didn't mean to commit yet